cli/dev/DevServer.ts

```diff
@@ -169,9 +175,37 @@ export class DevServer {
       },
     })
   }
-
   async stop() {
     this.httpServer?.close()
     this.eventsWatcher?.stop()
   }
+
+  private async handleInstallPackage(full_package_name: string) {
+    const postEvent = async (
+      event: "PACKAGE_INSTALLED" | "PACKAGE_INSTALL_FAILED",
+      message?: string,
+    ) => {
+      await this.fsKy.post("api/events/create", {
+        json: { event_type: event, ...(message ? { message } : {}) },
+        throwHttpErrors: false,
+      })
+    }
+
+    try {
+      console.log(`Installing package ${full_package_name}…`)
+      const cmd = `tsci add ${full_package_name}`
+
+      // Synchronously run the install
+      execSync(cmd, { stdio: "pipe" })
```

Comment:
Can you call the installation method directly? Going out to shell breaks across operating systems a lot so we want to do it as little as possible.

There should be a function already inplemented for it but if not you may need to break out the add method into a separate func